unlock_nft.go

```diff
@@ -35,3 +35,7 @@ func (r *NFTUnlock) Type() UnlockType {
 func (r *NFTUnlock) Size() int {
 	return NFTUnlockSize
 }
+
+func (r *NFTUnlock) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return workScoreStructure.WorkScoreEd25519Signature
```

Comment:
Btw, we never charge for these right? Because we never analyse inputs anyway? Maybe we should switch to charge costs if someone creates these things on the output side.

Reply:
I changed this now to just be a data factor for work score rather than charging for signature.